ci-operator/step-registry/openshift-qe/installer/bm/day2/lvms/openshift-qe-installer-bm-day2-lvms-commands.sh

```
@@ -0,0 +1,169 @@
+
+if oc get storagecluster -n openshift-storage 2>/dev/null | grep -q ocs-storagecluster; then
+  echo "ERROR: ODF StorageCluster detected. LVMS and ODF cannot share disks."
+  exit 1
+fi
+
+SSH_ARGS="-i ${CLUSTER_PROFILE_DIR}/jh_priv_ssh_key -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null"
+INNER_SSH_ARGS="-oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null"
+
+wipe_nvme() {
+  local bastion=$1
+  ssh ${SSH_ARGS} root@${bastion} '
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)